config/iblinds/ib2.xml

```diff
@@ -0,0 +1,66 @@
+<Product Revision="1" xmlns="https://github.com/OpenZWave/open-zwave">
+  <MetaData>
+    <MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/0287:0003:000D</MetaDataItem>
+    <MetaDataItem name="ProductPic">images/iblinds/iblinds.png</MetaDataItem>
+    <MetaDataItem id="0003" name="ZWProductPage" type="000D">https://products.z-wavealliance.org/products/2486/</MetaDataItem>
+    <MetaDataItem name="ResetDescription">The Device Reset Locally function is used to reset the device to fatory defalut.  The reset operation resets protocol data (HomeID, NodeID) as well as all application specific data to factory default values.vice Reset Locally
+NOTE: Please only use this function when the primary controller is missing or inoperable! 
+1.	Press the INC / EXC button 3
+Press the INC / EXC button 3 times within 5 seconds to enable the device reset function.
+2.	Wait 30 Seconds Press CLBR button
+After device is reset wait 30 seconds and press the CLBR button to reboot the device and make it ready for Z-Wave Inclusion.  </MetaDataItem>
+    <MetaDataItem id="0003" name="Identifier" type="000D">ib2.0</MetaDataItem>
+    <MetaDataItem name="ProductManual">https://products.z-wavealliance.org/ProductManual/File?folder=&amp;filename=MarketCertificationFiles/2486/iBlind_User_Guide.pdf</MetaDataItem>
+    <MetaDataItem name="Description">An intelligent Z-Wave blind motor that installs out of sight in existing Horizontal (2" and 2 1/2" slatted-style) blind headrails. Use iBlinds to tilt the slats open, closed or use the value slider for precise positioning of the slat tilt angle.</MetaDataItem>
+    <MetaDataItem name="ExclusionDescription">Follow the directions for your Z-Wave device to place it in the exclusion mode. 
+Press the exclude button once and wait for the device to be removed from the Z-Wave network. 
+ </MetaDataItem>
+    <MetaDataItem name="Name">iBlinds</MetaDataItem>
+    <MetaDataItem id="0003" name="FrequencyName" type="000D">U.S. / Canada / Mexico</MetaDataItem>
+    <MetaDataItem name="InclusionDescription">Follow the directions for your Z-Wave hub/controller to place it in the add for inclusion mode. 
+Press the inclusion button once and wait for the device to be discovered and joined to the Z-Wave network.  Follow the directions of your Z-Wave controller to uniquely identify your iblinds device.   (Ex: Bedroom iblind)
+Note: If the device is not joined to the network.  Press the CLBR button Wait 1 minutes and repeat steps 1 and 2 again.  If you continue to have trouble joining the iblinds motor to the Z-Wave network follow the Z-Wave network Exclusion process below.  
+</MetaDataItem>
+    <ChangeLog>
+      <Entry author="mike240se" date="10 Jul 2020" revision="1">Initial metadata</Entry>
+    </ChangeLog>
+  </MetaData>
+   <!-- Configuration Parameters -->
+	<CommandClass id="38">

```

Comment:
This Block Should not be required. Why did you include it?

Reply:
Just to change the help text.  the default talks about dimmers (this device should be a cover but due to a bug in the firmware or design decision its a dimmer).

For example by default it says "Increase the brightness of the device", i changed the help text to "Increase the blinds tilt"

Should I remove it?